### DISCUSSION TITLE
Hello for me to apply it says I need to "Finish my profile"

### DISCUSSION THREAD

**Discussion by ghost** [community/community#135931] [GitHub Education]
### Hi everyone,

Finishing my profile needs my full name and picture. I do not want private information being seen so easily on the internet. Is this necessary? I value my privacy

**Comment by burningmannn** [comment]
On GitHub, providing a full name and profile picture is not strictly necessary, but it can be beneficial for your online presence and collaboration with others. GitHub doesn't require your real full name. You can use a pseudonym or a nickname that you're comfortable with. If you're concerned about privacy you can upload a generic image or an avatar that doesn't reveal your identity.

**Comment by davevad93** [comment]
Hi @Robotix-47 ,

To apply for the `Student Developer Pack` your full name is required, but you don't need to upload a photo for your GitHub profile, just the name as it appears on your academic affiliation document.
